clients/java/src/test/java/io/camunda/zeebe/client/decision/rest/DecisionEvaluationRestTest.java

```diff
@@ -0,0 +1,338 @@
+import io.camunda.zeebe.client.protocol.rest.EvaluateDecisionRequest;
+import io.camunda.zeebe.client.protocol.rest.EvaluatedDecisionInputItem;
+import io.camunda.zeebe.client.protocol.rest.EvaluatedDecisionItem;
+import io.camunda.zeebe.client.protocol.rest.EvaluatedDecisionOutputItem;
+import io.camunda.zeebe.client.protocol.rest.MatchedDecisionRuleItem;
+import io.camunda.zeebe.client.protocol.rest.ProblemDetail;
+import io.camunda.zeebe.client.util.ClientRestTest;
+import io.camunda.zeebe.client.util.RestGatewayPaths;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DecisionEvaluationRestTest extends ClientRestTest {
+
+  private static final long DECISION_KEY = 123L;
+  private static final long DECISION_INSTANCE_KEY = 234L;
+  private static final String TENANT_ID = "foo";
+  private static final EvaluatedDecisionInputItem EVALUATED_INPUT =
+      new EvaluatedDecisionInputItem()
+          .inputId("input-id")
+          .inputName("input-name")
+          .inputValue("input-value");
+  private static final EvaluatedDecisionOutputItem EVALUATED_OUTPUT =
+      new EvaluatedDecisionOutputItem()
+          .outputId("output-id")
+          .outputName("output-name")
+          .outputValue("output-value");
+  private static final MatchedDecisionRuleItem MATCHED_RULE =
+      new MatchedDecisionRuleItem()
+          .ruleId("rule-id")
+          .ruleIndex(1)
+          .addEvaluatedOutputsItem(EVALUATED_OUTPUT);
+  private static final EvaluatedDecisionItem EVALUATED_DECISION =
+      new EvaluatedDecisionItem()
+          .decisionId("my-decision")
+          .decisionKey(DECISION_KEY)
+          .decisionName("My Decision")
+          .decisionVersion(1)
+          .decisionType("TABLE")
+          .decisionOutput("testOutput")
+          .tenantId(TENANT_ID)
+          .addEvaluatedInputsItem(EVALUATED_INPUT)
+          .addMatchedRulesItem(MATCHED_RULE);
+  private static final io.camunda.zeebe.client.protocol.rest.EvaluateDecisionResponse
+      EVALUATE_DECISION_RESPONSE =
+          new io.camunda.zeebe.client.protocol.rest.EvaluateDecisionResponse()
+              .decisionKey(DECISION_KEY)
+              .decisionId("my-decision")
+              .decisionName("My Decision")
+              .decisionVersion(1)
+              .decisionOutput("testOutput")
+              .decisionRequirementsId("decision-requirements-id")
+              .decisionRequirementsKey(124L)
+              .failedDecisionId("my-decision")
+              .failureMessage("decision-evaluation-failure")
+              .tenantId(TENANT_ID)
+              .decisionInstanceKey(DECISION_INSTANCE_KEY)
+              .addEvaluatedDecisionsItem(EVALUATED_DECISION);
+
+  //  @Test
+  //  public void shouldEvaluateStandaloneDecisionWithDecisionKey() {
```

Comment:
@tmetzke I need help to figure out what is the problem here, I didn't enough time to look into it.

Reply:
OK this answers one of my questions I had 😅

Reply:
Maybe @koevskinikola has some time tomorrow (timeboxed) to look at it? I am in a meeting all day 😭

Reply:
@nicpuppa the problem was [here](https://github.com/camunda/camunda/pull/22421/files#diff-0208fa97206611e26913a11cc99f69d67fdcca30a4eca01c6c19bb556d075325R77). You need to use a `.post` and it all works.